mne/channels/channels.py

```diff
@@ -559,8 +559,9 @@ def drop_channels(self, ch_names):
 
         Parameters
         ----------
-        ch_names : list
-            The list of channels to remove.
+        ch_names : str | list
+            Name of the channel to remove, or the list of channels to
+            remove.
```

Comment:
I prefer that you don't forget and get an error rather than silent pass...

with int it's not that dangerous as int is not iterable